website/src/layouts/base/header/HamburgerMenu.astro

```diff
@@ -0,0 +1,73 @@
+---
+import { getSession } from 'auth-astro/server';
+
+import HamburgerMenuItem from './HamburgerMenuItem.astro';
+import { getPathogenMegaMenuSections } from './getPathogenMegaMenuSections';
+import { LoginButton } from '../../../components/auth/LoginButton';
+import { Page } from '../../../types/pages';
+
+type Props = {
+    forceLoggedOutState: boolean;
+};
+
+const { forceLoggedOutState } = Astro.props;
+
+const pathogenMegaMenuSections = getPathogenMegaMenuSections();
+
+const session = await getSession(Astro.request);
+const showLoggedInState = !forceLoggedOutState && session?.user !== undefined;
+---
+
+<div>
```

Comment:
```suggestion
<nav>
```
So screen reader can see that this is a website navigation. (Has no other effect).